<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>PhoneDetailsBase</class>
  <widget class="QWidget" name="phoneDetails" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>403</width>
        <height>130</height>
      </rect>
    </property>
    <property name="sizePolicy" stdset="0" >
      <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
      </sizepolicy>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Form1</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item>
        <layout class="QGridLayout" >
          <property name="margin" stdset="0" >
            <number>0</number>
          </property>
          <property name="spacing" stdset="0" >
            <number>6</number>
          </property>
          <item row="0" column="2" >
            <widget class="QLabel" name="lblAreaCode" >
              <property name="text" stdset="0" >
                <string>Area code:</string>
              </property>
              <property name="alignment" stdset="0" >
                <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="0" >
            <widget class="QComboBox" name="cmbCountry" />
          </item>
          <item row="0" column="6" >
            <widget class="QLabel" name="lblExtension" >
              <property name="text" stdset="0" >
                <string>Extension:</string>
              </property>
              <property name="alignment" stdset="0" >
                <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="6" >
            <widget class="QLineEdit" name="edtExtension" >
              <property name="sizePolicy" stdset="0" >
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>0</vsizetype>
                </sizepolicy>
              </property>
            </widget>
          </item>
          <item row="1" column="2" >
            <widget class="QLineEdit" name="edtAreaCode" >
              <property name="sizePolicy" stdset="0" >
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>0</vsizetype>
                </sizepolicy>
              </property>
            </widget>
          </item>
          <item row="1" column="4" >
            <widget class="QLineEdit" name="edtNumber" />
          </item>
          <item row="1" column="1" >
            <widget class="QLabel" name="TextLabel2" >
              <property name="text" stdset="0" >
                <string>-</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="4" >
            <widget class="QLabel" name="TextLabel5" >
              <property name="text" stdset="0" >
                <string>Number:</string>
              </property>
              <property name="alignment" stdset="0" >
                <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="3" >
            <widget class="QLabel" name="TextLabel4" >
              <property name="text" stdset="0" >
                <string>-</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="0" >
            <widget class="QLabel" name="TextLabel1" >
              <property name="text" stdset="0" >
                <string>Country:</string>
              </property>
              <property name="alignment" stdset="0" >
                <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="5" >
            <widget class="QLabel" name="lblDivExtension" >
              <property name="text" stdset="0" >
                <string>-</string>
              </property>
              <property name="alignment" stdset="0" >
                <set>Qt::AlignVCenter|Qt::AlignLeft</set>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="Spacer3" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>QWidget</class>
      <extends>QWidget</extends>
      <header location="global" >QWidget</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
      <properties>
        <property type="String" >text</property>
      </properties>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>cmbCountry</tabstop>
    <tabstop>edtAreaCode</tabstop>
    <tabstop>edtNumber</tabstop>
    <tabstop>edtExtension</tabstop>
  </tabstops>
  <images>
    <image name="image0" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
  </images>
</ui>
